run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+if [ ! -f ./ircd.yaml ]; then
+  cp oragono.yaml ircd.yaml
+fi
+
+if [ ! -f ircd.db ]; then
+  ./oragono initdb
+fi
+
+if [ ! -f tls.crt ]; then
+  ./oragono mkcerts
+fi
+
+./oragono run
```

Comment:
This is interesting, I'm taking a look to see exactly what's going on here. If I switch this to be `exec oragono run`, then the container starts and then just closes two seconds later, with a log file equating to this:

```
2017-06-30T01:22:13Z : info : startup : Oragono v0.8.2 starting
2017-06-30T01:22:13Z : info : listeners : listening on :6667 using plaintext.
2017-06-30T01:22:13Z : info : listeners : listening on 127.0.0.1:6668 using plaintext.
2017-06-30T01:22:13Z : info : listeners : listening on [::1]:6668 using plaintext.
2017-06-30T01:22:13Z : info : listeners : listening on :6697 using TLS.
2017-06-30T01:22:13Z : info : listeners : websocket listening on :8080 using plaintext.
2017-06-30T01:22:13Z : info : startup : Server running
2017-06-30T01:22:15Z : info : shutdown : Oragono v0.8.2 exiting
```
The server doesn't _seem_ to get any signals (or at least doesn't output what it usually does when it gets killed by a signal), so I'm wondering why it's exiting on this.

To make+run the docker container I'm using these commands:
```
docker build -t oragono_test .
docker create -it -p 6667:6667 -p 6697:6697 --name oragono_test oragono_test
docker start oragono_test
```

Any obvious mistakes you can spot? :P